exec/r-python.py

```diff
@@ -56,7 +56,7 @@ def hivecommand_from_r(query = None, poll_interval = None , sample_size = None,
     verbose = False
     
     skip_ssl_cert_check = None
-    api_url = None
+    #api_url = None
```

Comment:
Ok will remove. This is from old code, which prevents the user from using any custom url. coz api_url = None, was followed by 
if api_url is None:
        api_url = "https://api.qubole.com/api/"